databrowser/src/domain/datasets/ui/tableView/toolBox/SearchAndFilterToolBoxPanel.vue

```diff
@@ -54,7 +54,7 @@ SPDX-License-Identifier: AGPL-3.0-or-later
           @filter="updateFilter(index, filter.operator, $event, true)"
         />
       </ToolBoxCardBody>
-      <ToolBoxCardBody>
+      <ToolBoxCardBody class="flex justify-between">
```

Comment:
Are these new classes needed? I tried it without them, the result seems to be the same. If they are not needed, please remove them.